numbat/src/tokenizer.rs

```diff
@@ -144,7 +144,7 @@ pub enum TokenKind {
     Eof,
 }
 
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
```

Comment:
Why this change? Even with your recent improvements, Token is still > 24 byte large(?). Maybe it's better to keep clones of this explicit, by not deriving Copy?

Reply:
Probably a vestige of me trying to fix a borrow checker error. Reverted.